### DISCUSSION TITLE
How do I connect automatic from a Database when open the program

### DISCUSSION THREAD

**Discussion by juanjosepablos** [textbrowser/biblioteq#329] [Q&A] [answered]
I have been reading the Administratio Guide, and the BiblioteQ PDF file, but I am unable to find how to define a Database and automatic open when BiblioteQ starts.

I had check the  biblioteq.conf the .Ini file as well, but still unable to figure out how to use the database WITHOUT first g to File -> Connect and use the Branch Selection.
I know that you can go to File -> Recent Sqlite Files -> 
But the idea is that if you only have 1 database. Configure somehow that you do not need to select.

**Comment by textbrowser** [comment]
This is what I do:

> /opt/biblioteq/biblioteq.sh --open-postgresql-database xbook_db

**Reply by juanjosepablos** [reply, marked as answer]
Thanks for the tip!!. 
I am testing with sqlite for now, and windows, so playing around I had come up with this:
`Biblioteq.exe --open-sqlite-database Data\BiblioteQ.sqlite`

Works!!!
